tests/TESTS/LinkLoss/device/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
```

Comment:
does not compile, missing mbed-os.lib

Reply:
I omitted .libs because we're not using `mbed export`. Are they required for cmake builds?

Reply:
does it work for you when you compile it with cmake?

Reply:
I added the .lib. However, mbedtools resolves ${MBED_CONFIG_PATH}  to .mbedbuild instead of cmake_build